### ISSUE TITLE
Arakoon fails to start with error: starts with 17571602, which ain't higher than 19229207

### ISSUE THREAD

**Issue by kinvaris** [openvstorage/arakoon#92]
A arakoon on our multiDC setup will not start anymore because of:

```
2016-06-30 15:29:41 469132 +0200 - perf-roub-01 - 117006/0 - arakoon - 14 - info - other: bKz9fJNRoh8ZGfe7
2016-06-30 15:29:41 469151 +0200 - perf-roub-01 - 117006/0 - arakoon - 15 - info - other: nhfE8ZP77taJwMBG
2016-06-30 15:29:41 469165 +0200 - perf-roub-01 - 117006/0 - arakoon - 16 - info - quorum_function gives 2 for 3
2016-06-30 15:29:41 469177 +0200 - perf-roub-01 - 117006/0 - arakoon - 17 - info - DAEMONIZATION=false
2016-06-30 15:29:41 469194 +0200 - perf-roub-01 - 117006/0 - arakoon - 18 - info - open_tlog_collection_and_store ~autofix:true
2016-06-30 15:29:41 469222 +0200 - perf-roub-01 - 117006/0 - arakoon - 19 - info - Unlinking "/mnt/hdd1/ovh/arakoon/hdd_roub_nsm_1/db/touch-117006"
2016-06-30 15:29:41 469296 +0200 - perf-roub-01 - 117006/0 - arakoon - 20 - info - Unlink of "/mnt/hdd1/ovh/arakoon/hdd_roub_nsm_1/db/touch-117006" failed with ENOENT
2016-06-30 15:29:41 470170 +0200 - perf-roub-01 - 117006/0 - arakoon - 21 - info - Unlinking "/mnt/hdd1/ovh/arakoon/hdd_roub_nsm_1/db/touch-117006"
2016-06-30 15:29:41 470278 +0200 - perf-roub-01 - 117006/0 - arakoon - 22 - info - Unlinking "/mnt/hdd1/ovh/arakoon/hdd_roub_nsm_1/tlogs/touch-117006"
2016-06-30 15:29:41 470333 +0200 - perf-roub-01 - 117006/0 - arakoon - 23 - info - Unlink of "/mnt/hdd1/ovh/arakoon/hdd_roub_nsm_1/tlogs/touch-117006" failed with ENOENT
2016-06-30 15:29:41 471065 +0200 - perf-roub-01 - 117006/0 - arakoon - 24 - info - Unlinking "/mnt/hdd1/ovh/arakoon/hdd_roub_nsm_1/tlogs/touch-117006"
2016-06-30 15:29:41 471170 +0200 - perf-roub-01 - 117006/0 - arakoon - 25 - info - Unlinking "/mnt/hdd1/ovh/arakoon/hdd_roub_nsm_1/tlogs/touch-117006"
2016-06-30 15:29:41 471209 +0200 - perf-roub-01 - 117006/0 - arakoon - 26 - info - Unlink of "/mnt/hdd1/ovh/arakoon/hdd_roub_nsm_1/tlogs/touch-117006" failed with ENOENT
2016-06-30 15:29:41 471796 +0200 - perf-roub-01 - 117006/0 - arakoon - 27 - info - Unlinking "/mnt/hdd1/ovh/arakoon/hdd_roub_nsm_1/tlogs/touch-117006"
2016-06-30 15:29:41 471873 +0200 - perf-roub-01 - 117006/0 - arakoon - 28 - info - Unlinking "/mnt/hdd1/ovh/arakoon/hdd_roub_nsm_1/tlogs/touch-117006"
2016-06-30 15:29:41 471904 +0200 - perf-roub-01 - 117006/0 - arakoon - 29 - info - Unlink of "/mnt/hdd1/ovh/arakoon/hdd_roub_nsm_1/tlogs/touch-117006" failed with ENOENT
2016-06-30 15:29:41 472517 +0200 - perf-roub-01 - 117006/0 - arakoon - 30 - info - Unlinking "/mnt/hdd1/ovh/arakoon/hdd_roub_nsm_1/tlogs/touch-117006"
2016-06-30 15:29:41 472609 +0200 - perf-roub-01 - 117006/0 - arakoon - 31 - info - File at /mnt/hdd1/ovh/arakoon/hdd_roub_nsm_1/tlogs/head.db does not exist
2016-06-30 15:32:57 751364 +0200 - perf-roub-01 - 117006/0 - arakoon - 32 - fatal - AUTOFIX: can't autofix this: (Failure;   "\"/mnt/hdd1/ovh/arakoon/hdd_roub_nsm_1/tlogs/48713.tlx\" starts with 17571602, which ain't highe
r than 19229207"); backtrace:; Raised at file "map.ml", line 117, characters 16-25; Called from file "src/core/lwt.ml", line 161, characters 4-40
2016-06-30 15:32:57 751399 +0200 - perf-roub-01 - 117006/0 - arakoon - 33 - fatal - going down: (Failure;   "\"/mnt/hdd1/ovh/arakoon/hdd_roub_nsm_1/tlogs/48713.tlx\" starts with 17571602, which ain't higher than 19229207")
; backtrace:; Raised at file "format.ml", line 185, characters 41-52; Called from file "format.m
l", line 427, characters 6-24
2016-06-30 15:32:57 751408 +0200 - perf-roub-01 - 117006/0 - arakoon - 34 - fatal - after pick
2016-06-30 15:32:57 753391 +0200 - perf-roub-01 - 117006/0 - arakoon - 35 - info - Crash log dumped
2016-06-30 15:32:57 895195 +0200 - perf-roub-01 - 127313/0 - arakoon - 0 - info - --- NODE STARTED ---
2016-06-30 15:32:57 895236 +0200 - perf-roub-01 - 127313/0 - arakoon - 1 - info - git_revision: tags/1.9.6-0-gea32319 
2016-06-30 15:32:57 895251 +0200 - perf-roub-01 - 127313/0 - arakoon - 2 - info - compile_time: 10/06/2016 13:21:09 UTC 
2016-06-30 15:32:57 895263 +0200 - perf-roub-01 - 127313/0 - arakoon - 3 - info - version: 1.9.6
2016-06-30 15:32:57 895277 +0200 - perf-roub-01 - 127313/0 - arakoon - 4 - info - NOFILE: 1024
```

**Comment by pploegaert** [comment]
Probably related to #84 - and not fixed in #88

**Comment by kinvaris** [comment]
It began here: 

```
2016-06-27 10:28:59 398128 +0200 - perf-roub-01 - 3185/0 - arakoon - 504535 - info - nhfE8ZP77taJwMBG is master
2016-06-27 10:29:40 629631 +0200 - perf-roub-01 - 3185/0 - arakoon - 504536 - info - TCP info for 178.33.60.66:26413: { state = 1; ca_state = 0; retransmits = 0; probes = 0; backoff = 0; options = 7; snd_wscale = 7; rcv_ws
cale = 7; rto = 204000; ato = 0; snd_mss = 1448; rcv_mss = 536; unacked = 0; sacked = 0; lost = 0; retrans = 0; fackets = 0; last_data_sent = 3128; last_ack_sent = 0; last_data_recv = 259016112; last_ack_recv = 3128; pmtu 
= 1500; rcv_ssthresh = 29200; rtt = 124; rttvar = 18; snd_ssthresh = 7; snd_cwnd = 7; advmss = 1448; reordering = 3; rcv_rtt = 0; rcv_space = 29200; total_retrans = 1 }
2016-06-27 10:29:40 629697 +0200 - perf-roub-01 - 3185/0 - arakoon - 504537 - info - TCP info for 178.33.62.163:26413: { state = 1; ca_state = 0; retransmits = 0; probes = 0; backoff = 0; options = 7; snd_wscale = 7; rcv_w
scale = 7; rto = 204000; ato = 0; snd_mss = 1448; rcv_mss = 536; unacked = 0; sacked = 0; lost = 0; retrans = 0; fackets = 0; last_data_sent = 1216; last_ack_sent = 0; last_data_recv = 259016112; last_ack_recv = 1216; pmtu = 1500; rcv_ssthresh = 29200; rtt = 112; rttvar = 11; snd_ssthresh = 7; snd_cwnd = 8; advmss = 1448; reordering = 3; rcv_rtt = 0; rcv_space = 29200; total_retrans = 2188 }
2016-06-27 10:29:59 419285 +0200 - perf-roub-01 - 3185/0 - arakoon - 504538 - info - nhfE8ZP77taJwMBG is master

2016-06-29 11:30:20 674199 +0200 - perf-roub-01 - 3122/0 - arakoon - 0 - info - --- NODE STARTED ---
2016-06-29 11:30:20 674242 +0200 - perf-roub-01 - 3122/0 - arakoon - 1 - info - git_revision: tags/1.9.6-0-gea32319 
2016-06-29 11:30:20 674256 +0200 - perf-roub-01 - 3122/0 - arakoon - 2 - info - compile_time: 10/06/2016 13:21:09 UTC 
2016-06-29 11:30:20 674269 +0200 - perf-roub-01 - 3122/0 - arakoon - 3 - info - version: 1.9.6
2016-06-29 11:30:20 674283 +0200 - perf-roub-01 - 3122/0 - arakoon - 4 - info - NOFILE: 1024

```

**Comment by domsj** [comment]
Can we get all the logs for all nodes in this cluster?
This was done with arakoon 1.9.7?

**Comment by domsj** [comment]
@kinvaris told me this was encountered with arakoon 1.9.6, which does not yet contain the fix from #88, so closing this issue.
Please update arakoon, fix/verify no arakoon nodes are still affected. (Will require manual intervention for the affected nodes.)
Create a new issue (or reopen an existing one) if you see this issue again afterwards.
